src/main/java/io/cdap/plugin/http/source/common/DataType.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```

Comment:
change copyright year to 2024

Reply:
Too many classes to change here, i wrote this, may be useful in future !
- This will change all copyright year in `.java` files that were `Added` in `latest commit` to `current year`  
```bash
git diff-tree --no-commit-id HEAD -r --diff-filter=A --name-only | grep -E '\.java$' | xargs -I {} sed -E -i "" "s/Copyright © [0-9]+ Cask Data, Inc./Copyright © $(date +%Y) Cask Data, Inc./" {}
```